Gopkg.toml

```diff
@@ -28,13 +28,13 @@ required = [
     "github.com/grpc-ecosystem/grpc-gateway/runtime"
 ]
 
-[[constraint]]
+[[override]]
```

Comment:
This is still relevant to my comment: https://github.com/ligato/cn-infra/pull/328#discussion_r211009794